lib/osgl/STLUtils.cpp

```diff
@@ -0,0 +1,51 @@
+#include <vapor/STLUtils.h>
```

Comment:
This file replicates much of what is in lib/common/STDUtils.cpp. Shouldn't everything be consolidated into a single STDUtils.cpp file?

Reply:
Ah yes, I was doing a lot of testing with OpenGL and Conda so it was easier to have osgl be standalone from vapor. I've removed osgl/STLUtils and linked it against common/STLUtils.